crud.py

```diff
@@ -0,0 +1,44 @@
+from sqlalchemy.orm import Session
+
+from db import models
+from schemas import AuthorBaseCreate, BookBaseCreate
+
+
+def get_authors(db: Session):
+    return db.query(models.DBAuthor).all()
+
+
+def get_author_by_name(db: Session, name: str):
+    return db.query(models.DBAuthor).filter(models.DBAuthor.name == name).first()
+
+
+def create_author(db: Session, author: AuthorBaseCreate):
+    db_author = models.DBAuthor(
+        name=author.name,
+        bio=author.bio,
+    )
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+    return db_author
+
+
+def get_books(db: Session):
+    return db.query(models.DBBook).all()
+
+
+def get_book_by_title(db: Session, title: str):
+    return db.query(models.DBBook).filter(models.DBBook.title == title).first()
```

Comment:
The task requires filtering books by author ID. Consider adding a function that retrieves books based on the `author_id` parameter.